main.c

```diff
@@ -299,9 +299,13 @@ STATIC bool maybe_run_list(const char *const *filenames, size_t n_filenames) {
     if (_current_executing_filename == NULL) {
         return false;
     }
-    mp_hal_stdout_tx_str(line_clear);
+
+    // This function is used for `boot.py` and is then logged into `boot_out.txt`.
+    // We do not want the line clear to be logged.
+    // The function `serial_write` is the only function that isn't logged into the file.
+    serial_write(line_clear);
     mp_hal_stdout_tx_str(_current_executing_filename);
-    serial_write_compressed(MP_ERROR_TEXT(" output:\n"));
+    mp_hal_stdout_tx_str(" output:\n");
```

Comment:
Don't make this change. This is a translatable and compressed string. (Though no one has translated it yet.)

The comment you added just is helpful -- thanks.

Reply:
Alright, reverted the commit.